egs2/asvspoof/spk1/local/cm_data_prep.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
[question] Where do we need this? 
Do we need the train and the validation sets as well?

Reply:
No. I did not use this. I thought perhaps someone using the recipe might want to do some in-domain finetuning. I could take it out if it is not needed.